test/test_owners.csv

```diff
@@ -816,7 +816,7 @@ k8s.io/kubernetes/pkg/volume/fc,rrati,0,
 k8s.io/kubernetes/pkg/volume/flexvolume,Q-Lee,1,
 k8s.io/kubernetes/pkg/volume/flocker,jbeda,1,
 k8s.io/kubernetes/pkg/volume/gcepd,saad-ali,0,
-k8s.io/kubernetes/pkg/volume/git_repo,davidopp,1,
+k8s.io/kubernetes/pkg/volume/gitrepo,davidopp,1,
```

Comment:
Random question for @kubernetes/sig-testing , is this file even still maintained/used, or has it been obsoleted by owners files?  Looking at the storage related entries here, the owners are all mostly out of date...

Reply:
I don't think so, @spiffxp

Reply:
it's used, not very well, but I'd rather see its usages replaced/removed before removing the file (ref: https://github.com/kubernetes/kubernetes/pull/81544#issuecomment-522256127)